src/transformers/models/opt/modeling_opt.py

```diff
@@ -638,7 +633,7 @@ def forward(
 
         # embed positions
         if attention_mask is None:
-            attention_mask = torch.ones(inputs_embeds.shape[:2], dtype=torch.bool, device=inputs_embeds.device)
```

Comment:
Can we revert this? We don't infer the attention_mask from padding tokens in the input_ids 

cc @sgugger @LysandreJik  think we should have better docs (maybe under trouble shooting or even directly under `input_ids` that we don't automatically infer the attention_mask from padding tokens, but that the user needs to pass an attention_mask.
Also happy to open a PR that throws a warning if `input_ids` are passed with a padding token but no attention_mask is provided (think we discussed this a while back, but it has been forgotten). We could do the 1 warning only for multiple forward passes approach

Reply:
Yes, it was a temporary fix, think this should not be an issue